### PR TITLE
Send cni-conf-file to cni relation

### PR DESCRIPTION
**Pull request by Cynerva**
Partial fix for https://bugs.launchpad.net/charm-kubernetes-master/+bug/1861709

Please be careful merging this, as it's mutually dependent on the work of other PRs in that issue. They will need to be merged together.

This updates flannel to send cni-conf-file to the cni relation.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+  - "3.5"
```
Comment (kwmonroe) [inline .travis.yml:3]: include 3.6 and 3.7 so we cover default python versions on xenial/bionic/focal

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,21 @@
+[tox]
+skipsdist = True
+envlist = lint,py3
+
+[tox:travis]
+3.5: lint,py3
```
Comment (kwmonroe) [inline tox.ini:6]: include 3.6 and 3.7 so we cover default python versions on xenial/bionic/focal

**Review by kwmonroe — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (Cynerva) [reply .travis.yml:3]: done

File: tox.ini (hunk 2 above)
Comment (Cynerva) [reply tox.ini:6]: done

**Review by tvansteenburgh — APPROVED** [review]

**Pull request merged**